src/werkzeug/_internal.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
@teymour-aldridge We can't import this and support Python 3.6, so for the while these can't be present.

Reply:
Yeah, I realise that this is a problem and would break < 3.7 compatibility. The issue as I see it is that without these we'd have to use quotes for everything and then at a later stage we'd take all the quotes out again.
Given that `PyPy` also looks like it doesn't support forward referencing, I think that's probably the best solution.
Python 3.6 is getting pretty old now (4 years) and currently only security fixes for it are being published ([reference](https://www.python.org/dev/peps/pep-0494/#id23)), so I guess that at some point it will become possible to do this.

Reply:
I've now removed the these.